src/sql-parser/src/parser.rs

```diff
@@ -3868,9 +3868,18 @@ impl<'a> Parser<'a> {
 
     fn parse_cluster_replica_name(&mut self) -> Result<QualifiedReplica, ParserError> {
         let cluster = self.parse_identifier()?;
-        self.expect_token(&Token::Dot)?;
-        let replica = self.parse_identifier()?;
-        Ok(QualifiedReplica { cluster, replica })
+        if !self.consume_token(&Token::Dot) {
+            self.prev_token();
+
+            self.expected(
+                self.peek_pos(),
+                "cluster_identifier.replica_identifier",
```

Comment:
Might be a bit cleaner to quote the identifiers like `"\"cluster_identifier\".\"replica_identifier\""`. I'm not really sure what the standard is here.